code/modules/mob/living/simple_animal/abnormality/aleph/melting_love.dm

```diff
@@ -301,3 +323,134 @@
 	melee_damage_lower = 35
 	melee_damage_upper = 40
 	spawn_sound = 'sound/abnormalities/meltinglove/pawn_big_convert.ogg'
+	statuschance = 75
+
+//Slime trails
+/obj/effect/decal/cleanable/melty_slime
+	name = "Slime"
+	desc = "It looks corrosive."
+	icon = 'ModularTegustation/Teguicons/tegu_effects.dmi'
+	icon_state = "melty_slime3"
+	random_icon_states = list("melty_slime3")
+	mergeable_decal = TRUE
+	var/duration = 30 SECONDS
+	var/state = 3
+	var/timer1
+	var/timer2
+	var/list/slime_types = list(
+		/mob/living/simple_animal/hostile/abnormality/melting_love,
+		/mob/living/simple_animal/hostile/slime/big,
+		/mob/living/simple_animal/hostile/slime
+	)
+
+/obj/effect/decal/cleanable/melty_slime/Initialize(mapload, list/datum/disease/diseases)
+	. = ..()
+	START_PROCESSING(SSobj, src)
+	duration += world.time
+	timer1 = addtimer(CALLBACK(src, PROC_REF(Reduce)), 10 SECONDS, TIMER_STOPPABLE)
+	timer2 = addtimer(CALLBACK(src, PROC_REF(Reduce)), 20 SECONDS, TIMER_STOPPABLE)
+
+/obj/effect/decal/cleanable/melty_slime/proc/Refresh()
+	icon_state = "melty_slime3"
+	duration = 30 SECONDS
+	if(timer1)
+		deltimer(timer1)
+		timer1 = null
+	if(timer2)
+		deltimer(timer2)
+		timer2 = null
+	timer1 = addtimer(CALLBACK(src, PROC_REF(Reduce)), 10 SECONDS, TIMER_STOPPABLE)
+	timer2 = addtimer(CALLBACK(src, PROC_REF(Reduce)), 20 SECONDS, TIMER_STOPPABLE)
+
+
+/obj/effect/decal/cleanable/melty_slime/proc/Reduce()
+	state -= 1
+	icon_state = "melty_slime[state]"
+	update_icon()
```

Comment:
Have you tried using `animate()` to animate from one icon to another? It might look better?

Reply:
I did, I had a bunch of issues with it like the animation not resetting properly and problems with matrixes and such. Using animate() would make a lot more sense if I had icons meant to be resized.